spec/i18n_spec.rb

```diff
@@ -106,6 +106,7 @@ class BaseTask
         { key: 'idv.failure.setup.fail_html', locales: %i[zh] },
         { key: 'idv.failure.verify.exit', locales: %i[zh] },
         { key: 'in_person_proofing.form.state_id.state_id_number_florida_hint_html', locales: %i[zh] },
+        { key: 'in_person_proofing.process.eipp_bring_id.image_alt_text', locales: %i[fr es zh] },
```

Comment:
Is there a ticket to follow-up on this?

Reply:
Thanks for calling this out. For items that are still not translated entirely, as in this example we should 1. Request another translation and 2. Make updates accordingly (and then remove from this list.) I will create a ticket for this and link it in this thread.

Reply:
@aduth - After talking with @rutvigupta-design I learned that DOS is treating Real ID as a proper noun. As a result the translations include Real. I moved `in_person_proofing.process.eipp_bring_id.image_alt_text` above the `#need to be fixed` line and added a comment about it. I also just alphabetized the section. This will also go through the LQA (Language Quality Assurance) process again. (A screenshot will be sent off to get approval.) 

Knowing this information, with my change, I will not be opening a ticket. Let me know if you have opposition so we can come to something agreeable.  

Thank you

See commit d31b04dcc37b6aabc9fbe554bde1b947d601c823

Screenshot of translations
![Screenshot 2024-06-06 at 12 43 18 PM](https://github.com/18F/identity-idp/assets/125507397/3e2a0ed9-f7d6-42bd-a101-c3cb3f2a5644)

Reply:
No concern with that 👍 Appreciate the added code comment as well.